Gordon360/Models/ViewModels/RecIM/MatchStatsPatchViewModel.cs

```diff
@@ -3,10 +3,9 @@
 
 namespace Gordon360.Models.ViewModels.RecIM
 {
-    public class MatchTeamPatchViewModel
+    public class MatchStatsPatchViewModel
     {
         public int TeamID { get; set; }
```

Comment:
consider carefully what you want to allow to be updated here and in the service/controller methods. I'm confused as to why you would want to allow a TeamID to be updated for the stats object. I would imagine that once a stats object is created for a team, it represents that team's stats for the match in question. If you were to change the teams assigned to a match, it would make sense to delete (or change the status of) the existing stats objects and create new one/s for the new team/s.